tensorboard/plugins/scalar/scalars_plugin.py

```diff
@@ -34,6 +34,7 @@
 from tensorboard.backend import http_util
 from tensorboard.compat import tf
 from tensorboard.plugins import base_plugin
+from tensorboard.plugins.profile import end_to_end_util
```

Comment:
The scalar plugin cannot have a code dependency on the profile plugin.
Writing scalar summaries to a specially named run is not an appropriate
way to manage profile data.

Reply:
Fixed.